Automattic/wp-calypso
# Stats: Restore the most popular post with the data of the last 365 days

Related to #74008 partially

## Proposed Changes

* Change the `top posts` data period from `7 days` to `1 year` as proposed in the related discussion (`pejTkB-nM-p2`).
* Merge `LatestPostCard` and `MostPopularPostCard` into `PostCardsGroup` for applying the `DotPager` component.
* Remove feature flags `stats/latest-post-stats` and `stats/most-popular-post`.

## Testing Instructions



* Spin up the change with the Live Branch.
* Navigate to the Stats `Insights` page.
* Ensure the card of the `Most popular post` is displayed with the data of the last 365 days.
* Ensure the mobile layout of cards aligns with the design.

|PC|Mobile|
|-|-|
|<img width="1276" alt="截圖 2023-03-17 上午1 29 50" src="https://user-images.githubusercontent.com/6869813/225704244-13bc87f9-a84a-497e-ada1-fb6a09269836.png">|<img width="457" alt="截圖 2023-03-17 上午1 30 26" src="https://user-images.githubusercontent.com/6869813/225704286-42c4aa5f-7a18-4107-b833-6e66108f0ee4.png">|

## Pre-merge Checklist
